src/Uno.UI.RemoteControl.Messaging/IDEChannel/IdeResultMessage.cs

```diff
@@ -0,0 +1,5 @@
+﻿#nullable enable
+
+namespace Uno.UI.RemoteControl.Messaging.IdeChannel;
+
+public record IdeResultMessage(long IdeCorrelationId, Result Result) : IdeMessage(WellKnownScopes.HotReload);
```

Comment:
Shouldn't this be similar to the IdeMessageWithCorrelationId in that it accepts and forwards to the base IdeMessage, a Scope (string) parameter?